.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2.1
+jobs:
+  release:
+    docker:
+      - image: circleci/golang:1.13
+    steps:
+      - checkout
+      - run: curl -sL https://git.io/goreleaser | bash
+  build:
+    docker:
+      - image: circleci/golang:1.13
+        environment:
+          GO111MODULE: "on"
+          TEST_RESULTS: /tmp/test-results
+    working_directory: /go/src/github.com/sonatype-nexus-community/nancy
+    steps:
+      - checkout
+      - run: mkdir -p $TEST_RESULTS
+      - restore_cache:
+          keys:
+            - go-mod-v1-{{ checksum "go.sum" }}
+      - run: go mod download
+      - run: go get -u github.com/jstemmer/go-junit-report
+      - run: go test ./... -v 2>&1 | go-junit-report > $TEST_RESULTS/report.xml
+      - run: go build 
+      - run: cd testdata/dep && ../../nancy Gopkg.lock && cd -
+      - run: ./nancy go.sum
+      - run: go list -m all | ./nancy
+      - save_cache:
+          key: go-mod-v1-{{ checksum "go.sum" }}
+          paths:
+            - "/go/pkg/mod"
+      - store_artifacts:
+          path: /tmp/test-results
+          destination: raw-test-output
+      - store_test_results:
+          path: /tmp/test-results
```

Comment:
can/should you use `TEST_RESULTS` here?

Reply:
Yeah or does CircleCI already magically do things with it?? 

My current concern is *when* a test fails in CI currently it just shows nothing in output.
![image](https://user-images.githubusercontent.com/1099906/75119046-78f4c880-564d-11ea-8693-dd4d685e8f13.png)

But maybe thats fine cause CircleCI has this tab?? Which maybe has more details when something fails?? #circleCiNewb
![image](https://user-images.githubusercontent.com/1099906/75119063-9cb80e80-564d-11ea-9aa5-75c03feedaac.png)

Reply:
I forgot why I did this, just kinda muscle memory at this point. CircleCI does not support an environment variable here, for some whatever reason!